Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/eris/build
+FROM quay.io/monax/build
```

Comment:
can be versioned: https://github.com/monax/cli/pull/1295/files#diff-0551cf25c62a749fe9b04d036a1ef478R44